<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreatEvent</class>
 <widget class="QDialog" name="CreatEvent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CreatEvent</string>
  </property>
  <widget class="QPlainTextEdit" name="plainTextEdit_Show">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>471</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>产生人工信号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton__SetEvent">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>260</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>有信号(SetEvent)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Reset">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>260</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>无信号(Reset)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>320</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>产生自动信号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_PulseEvent">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>320</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>脉冲(PulseEvent)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Wait">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>320</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>等待(Wait)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Close">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>260</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭(Close)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>信号个数</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_EventNum">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>189</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ThreadWaitNum">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>190</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>201</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>等待线程</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_StartThread">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>440</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>开启新线程</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
